### DISCUSSION TITLE
Global search with Spotlight

### DISCUSSION THREAD

**Discussion by sald19** [filamentphp/filament#11617] [Help]
### Package

Panel builder

### Package Version

v3.2.28

### How can we help you?

Hello, first of all I want to thank the entire team that makes filament possible, thanks guys.

I want to ask if there is a way to use global search in a spotlight; with the same look and feel as the spotlight plugin
https://filamentphp.com/plugins/pxlrbt-spotlight

![pxlrbt-spotlight](https://github.com/filamentphp/filament/assets/1674971/22c405b9-b8b4-4dde-b334-db4a698c2361)

**Comment by leandrocfe** [comment]
Global search in a spotlight? What does it mean?

**Reply by sald19** [reply]
Hi @leandrocfe 👋🏽

I mean the global search looks like this

from this 
![CleanShot 2024-02-29 at 10 15 24](https://github.com/filamentphp/filament/assets/1674971/56121e50-0f32-43e2-ba1f-2c8bb6eb7587)

 to this
![CleanShot 2024-02-29 at 10 12 59](https://github.com/filamentphp/filament/assets/1674971/8ae7bc04-a527-4f3e-b949-74e74a7e2757)

**Reply by MhdTarhini** [reply]
Hey, I'm facing the same problem with pxlrbt-spotlight where records are not showing. Did you solve it ? Thank you in advance

**Comment by maukoese** [comment]
There's a package/plugin [that's close](https://filamentphp.com/plugins/charrafimed-global-search-modal)
